src/components/profile/Profile.js

```diff
@@ -25,16 +25,21 @@ const ProfileWrapper = props => {
     screenProps.push(`ViewAvatar`)
   }
 
+  const handlePrivacyPress = useCallback(() => screenProps.push(`ProfilePrivacy`), [screenProps])
```

Comment:
good
But there's also a non-wrapped handleAvatarPress function just above 
Could you please fix it too. I'm understand that it's probably not your code but it's just an small fix
For the future - if you see some small fix/optimizations try to introduce it at least in the whole file you're working with. It'll not take so much time but will make an app a bit better

Reply:
@serdiukov-o-nordwhale 
Sure.
I pushed a fix for handleAvatarPress.